plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -2532,7 +2532,7 @@ private void testConcurrentInsertsSelectingFromTheSameTable(boolean partitioned)
                     .collect(toImmutableList());
 
             long successfulInsertsCount = futures.stream()
-                    .map(future -> tryGetFutureValue(future, 20, SECONDS).orElseThrow(() -> new RuntimeException("Wait timed out")))
+                    .map(future -> tryGetFutureValue(future, 60, SECONDS).orElseThrow(() -> new RuntimeException("Wait timed out")))
```

Comment:
it's too generous. Can we go 30 s first?

Reply:
Decreased to 40s